dashboard/app/controllers/omniauth_callbacks_controller.rb

```diff
@@ -462,15 +462,15 @@ def silent_takeover(oauth_user, auth_hash)
           oauth_refresh_token: auth_hash.credentials&.refresh_token
         )
       end
-    rescue => err
+    rescue => e
       error_class = lookup_user.migrated? ?
         'Failed to create AuthenticationOption during silent takeover' :
         'Failed to update User during silent takeover'
       # This should never happen if other logic is working correctly, so notify
       # This can happen if the account being taken over is already invalid
       Honeybadger.notify(
         error_class: error_class,
-        error_message: err.to_s,
+        error_message: e.to_s,
```

Comment:
```suggestion
        error_message: e.message,
```

Slightly more explicit than `to_s`